assets/js/DataGrid.test.js

```diff
@@ -28,6 +28,9 @@ describe('DataGrid tests,', () => {
     ]
   };
 
+  var sortDirectionMock = jest.fn()
```

Comment:
This is OK, as long as no test checks to see if the function mock was called (the same mock will be used in all places). If that becomes a problem we can put the initialization inside `beforeEach`